docs/guidance/publish/change_history.md

```diff
@@ -1,14 +1,18 @@
 # Change history
 
-The following example shows how to publish the full history of a contracting process using releases and records. Each subsection refers to a stage or event in the contracting process lifetime.
+The following example shows how to publish a contracting process' data as it unfolds. Releases are built and published for each event in the process. Records are updated for each new release available.
+
+Each subsection refers to a stage or event in the contracting process lifetime.
 
 Refer to the [change history](../../getting_started/change_history) guidance for an introduction to the concepts involved.
 
 ### Planning
 
-In order to prepare the market for the tender for cycle lane improvements which is planned for later in the year, the London Borough of Barnet publishes a 'notice of planned procurement' (also known as 'prior information notice' or 'future opportunity notice').
+The London Borough of Barnet has a planned tender for later in the year. To prepare the market they publish a *notice of planning procurement*. This is also known as *prior information notice*, or *future opportunity notice*.
+
+The publisher creates an OCDS release to represent this notice. The release uses the 'planning' tag and includes data of the planned procurement. The `tender` section holds the estimated values, as the issue date. The section also uses the 'planned' status to state these are planned values.
 
-The notice is represented in OCDS as a release with the 'planning' tag and data on the planned procurement, such as the estimated date the tender will be issued, is provided in the `tender` section with `.status` set to 'planned'. The `releases` list in the record for the contracting process is updated to include the ‘planning’ release, the only one that exists for the process at the moment. The compiled and versioned releases inside the record contain the latest information, which should be very similar to the 'planning' release. Compare the planning release and the record using the box below.
+The publisher also creates an OCDS Record for the starting process. The releases list includes the new and only release so far. The compiled and versioned releases are also created for the record. Since there is only one release, the compiled version should look very much like this release. Compare the planning release and the record using the box below.
```

Comment:
```suggestion
The publisher also creates an OCDS record for the new contracting process. The releases list includes the new and only release so far. The compiled and versioned releases are also created for the record. Since there is only one release, the compiled version should look very much like this release. Compare the planning release and the record using the box below.
```